frontend/src/css/PublicProfile.styl

```diff
@@ -1,3 +1,17 @@
+// à placer dans un doc dédié au popup call
+
```

Comment:
This should probably be done indeed. But I'm fine with leaving this in PublicProfile, then just remove this comment.
```suggestion
```